packages/cli/src/index.ts

```diff
@@ -18,25 +19,9 @@ updateNotifier
 
 program
   .version(packageJSON.version, undefined, 'display E2B CLI version')
-  .addOption(new commander.Option('-cmd2json').hideHelp())
-  .on('option:-cmd2json', () => {
-    process.stdout.write(
-      JSON.stringify(
-        program.commands
-          .map((x: any) => ({
-            command: x.name(),
-            description: stripAnsi.default(x.description()),
-            options: x.options.map((y: any) => ({
-              flags: y.flags,
-              description: stripAnsi.default(y.description),
-              defaultValue: y.defaultValue,
-            })),
-          }))
-          .sort((row1: any, row2: any) =>
-            row1.name().localeCompare(row2.name()),
-          ),
-      ),
-    )
+  .addOption(new commander.Option('-cmd2md').hideHelp())
```

Comment:
I am just thinking, would it make sense to have this only in dev and not in prod version?